angular-workspace/projects/example-client-app/src/app/customapp/customapp.component.html

```diff
@@ -197,6 +197,12 @@
                 >
                     <nimble-icon-check title="String 1"></nimble-icon-check>
                 </nimble-table-column-text>
+                <nimble-table-column-anchor
```

Comment:
Thoughts on demonstrating the router link with href directive in the example app? I know we don't demonstrate every feature from here but that configuration seems interesting enough to add to me.

Reply:
Want to avoid adding additional routes to separate pages to keep the example angular and blazor apps similar: https://github.com/ni/nimble/pull/1184#discussion_r1169368423

Maybe could use the router to [add a query parameter to the url](https://angular.io/api/router/RouterLink#setting-and-handling-query-params-and-fragments) and render it in a text box or something

Reply:
Did as suggested.